### ISSUE TITLE
Can't copy values from results window

### ISSUE THREAD

**Issue by nstansby** [Huachao/vscode-restclient#647]
- REST Client Version: 0.24.1
- VSCode Version: 1.47.2
- OS Version: Win 10 Pro v2004 build 19041.388

Steps to Reproduce:

1. Send any request that returns a response body
2. Try to select a section of the response text and copy it to the clipboard

Tried this with ctrl-C as well as right-mouse-button.  Neither works.

**Comment by gertjvr** [comment]
Similar, found out the copy has moved to hidden buttons at the top right, I personally prefer being able to copy straight from the output window

![image](https://user-images.githubusercontent.com/1691062/88493537-f8b80200-cff4-11ea-9c54-964d5b0bc538.png)

**Comment by nstansby** [comment]
@gertjvr Yes I saw that, but often I just want to copy a small subsection of the response.  I've been using that as a workaround (copy the entire response to a new file, then copy and paste from there), but that's far from ideal.

@Huachao A bit of additional info: It seems this is transient... I just did a query that returned a large JSON response, was able to copy a few values out of it and then it stopped working.  Restart VS code, re-run the query, and I can copy one or two more values... then same problem.
